### PR TITLE
Initial setup of ION Q-and-A including faq.md of Henry Tsai

### PR DESCRIPTION
**Pull request by henkvancann**

### CODE REVIEW

File: docs/Q-and-A.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+## ***Q: What would be the quickest way for me to learn more about 'unpublished' DIDs? 
+_Do you happen to know of a link to any documentation about that?_
+
+This one class invocation will instantly create you an unpublished DID: [here](https://t.co/eBY4wR2kTk?amp=1)\
+(_@csuwildcat_)
+
+## ***Q: Any plans to make this work on top of bitcoin full nodes?; on top of projects like [Umbrel](https://blocktalk.co/2021/03/25/how-to-set-up-full-bitcoin-node-umbrel/)?
+I haven't had time to sort out getting it packaged for this. Would love some help with that from the community, else it will probably take a couple months before we can dig out.\
+(_@csuwildcat_)
```
Comment (AaronDewes) [inline docs/Q-and-A.md:229]: You could mention that @lounes_kmt is working on an Umbrel implementation: https://github.com/getumbrel/umbrel/issues/560

Comment (henkvancann) [reply docs/Q-and-A.md:246]: done, thx for tip

Comment (AaronDewes) [reply docs/Q-and-A.md:246]: Can you revert that? It shouldn't be announced officially, sorry.

Comment (henkvancann) [reply docs/Q-and-A.md:246]: yes of course, and no prob yet, it's still only on my repo

Comment (henkvancann) [reply docs/Q-and-A.md:246]: has gone

**Review by csuwildcat — APPROVED** [review]

**Pull request merged**